go.mod

```diff
@@ -1,7 +1,3 @@
-module github.com/golang-jwt/jwt/v4
+module github.com/golang-jwt/jwt/v5
```

Comment:
In order for users to continue fetching the /v4 module, and for us to patch issues in the v4.x.y version .. should we also have a `v4` branch that branches off from 9358574a7a1a2c8d644f22b6e8de627ba85c58d0 (or whatever the last commit before they diverge)?

Reply:
We can probably do that, yes

Reply:
I published a `v4.5.0` tag for 9358574a and created a separate [v4 branch ](https://github.com/golang-jwt/jwt/tree/v4).